liblangutil/SourceReferenceFormatter.h

```diff
@@ -51,7 +52,17 @@ class SourceReferenceFormatter
 	virtual void printExceptionInformation(SourceReferenceExtractor::Message const& _msg);
 
 	virtual void printSourceLocation(SourceLocation const* _location);
-	virtual void printExceptionInformation(dev::Exception const& _error, std::string const& _category);
+	virtual void printExceptionInformation(dev::Exception const& _exception, std::string const& _category);
+
+	/// When we don't care detailed type of error, we can use these shortcuts to format Error
```

Comment:
Hm, I don't quite get the meaning of this comment. Is it needed at all?

Reply:
It is not needed. I mean, for example, instead of printing `DeclarationError` or `ParserError`, we simply print `Error`.